app/components/NarrationsContinueView.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { Fragment, Path } from "@/types/videosAndFilms";
+import "@radix-ui/themes/styles.css";
+import { Flex, Text } from "@radix-ui/themes";
+import NarrationsContinueButton from "@/ui/NarrationsContinueButton";
+import Countdown from "@/ui/countDown";
+import { useEffect, useState } from "react";
+import VideoPlayer from "@/components/admin/videos/VideoPlayer";
```

Comment:
@NikolaDziwusz lets use the FilmPlayer from `app/components/films/FilmPlayer.tsx`